example/example-reader.py

```diff
@@ -0,0 +1,59 @@
+import copclib
+
+
+if __name__ == "__main__":
+
+    # Create a reader object
+    reader = copclib.FileReader("../build/test/data/autzen-classified.copc.laz")
+
+    # We can get the CopcData struct
+    copc_vlr = reader.GetCopcHeader()
+    print("CopcData: ")
+    print("\tSpan: %d" % copc_vlr.span)
+    print("\tRoot Offset: %d" % copc_vlr.root_hier_offset)
+    print("\tRoot Size: %d" % copc_vlr.root_hier_size)
+
+    # Get the Las Header
+    las_header = reader.GetLasHeader()
+    print("\nLas Header:")
+    print(
+        "\tPoint Format: %d\n\tPoint Count: %d"
+        % (las_header.point_format_id, las_header.point_count)
+    )
+
+    # Get the WKT string
+    print("WKT: %s\n" % reader.GetWkt())
+
+    load_key = copclib.VoxelKey(4, 11, 9, 0)
+
+    # FindNode will automatically load the minimum pages needed
+    # to find the key you request
+    node = reader.FindNode(load_key)
+    # If FindNode can't find the node, it will return an "invalid" node:
+    if not node.IsValid():
+        quit()
```

Comment:
return?

Reply:
Once it's defined as `ExampleReader()` function I will change to return yep